sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -202,6 +202,16 @@ export interface KeyProperties {
 // @public
 export type KeyType = "EC" | "EC-HSM" | "RSA" | "RSA-HSM" | "oct";
 
+// @public
+export class KeyVaultIdentifier implements ParsedKeyVaultIdentifier {
```

Comment:
I don't think we want to be exposing the same name across all packages. We should talk with the architects what they think. @schaabs suggested we could do package-specific names like `KeyVaultCertificateIdentifier` that could even assert the collection is, for example, "certificates" or "deletedcertificates".

Reply:
Gotcha! That makes sense.